src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -167,7 +187,7 @@ public AttachCaseCallbackService(CcdApi ccdApi) {
         } catch (Exception exc) {
             log.error(
                 "Error attaching ER {} in {} to case {}",
-                event.exceptionRecordId,
+                event.exceptionRecord.getId(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Potential violation of Law of Demeter (static property access)](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4558409)